scanner.cs

```diff
@@ -3,66 +3,176 @@
 using System.Collections.Generic;
 
 public class Scanner {
-	private int curByte = 0;
-	private byte[] __bytes;
-	private List<Tuple<string, TOKENS>> __tokens;
+    private int curByte = 0;
+    private int column = 1;
+    private int line = 1;
```

Comment:
Yeah, I'm not sure why the scanner needs those. Who wants to know where the scanner is besides the scanner? I think those are actually meant for tokens, see my comment on ticket 10 on the scrum board.